### ISSUE TITLE
Command not found: xcsync

### ISSUE THREAD

**Issue by DgkanatsiosMasoutis** [microsoft/vscode-dotnettools#1605]
I get this error when i try to run xcsync generate on CLI VS code:

 **zsh: command not found: xcsync**

I checked if the xcsync tool is installed and i get "Tool 'dotnet-xcsync' is already installed.". I can't find why is this happening.

**Comment by davidortinau** [comment]
@DgkanatsiosMasoutis is it installed globally? This is needed at present.

On your Mac:

```console
dotnet tool install --global dotnet-xcsync --version 0.1.0-preview.24509.2
```

https://www.nuget.org/packages/dotnet-xcsync/0.1.0-preview.24509.2

Perhaps uninstall what you have and install for that command.

**Comment by DgkanatsiosMasoutis** [comment]
@davidortinau yes it is installed globally. I found a that it is not enabled every time and i have to enable it with a command that only works for the current session but it works fine. Also i have another issue. I want to create a widget within my Xcode project and then transfer it to my .net MAUI solution but it tells me i cannot. Do i need something specific to make it work or it is just my code? Thanks in advance!

**Comment by mcumming** [comment]
> [@davidortinau](https://github.com/davidortinau) yes it is installed globally. I found a that it is not enabled every time and i have to enable it with a command that only works for the current session but it works fine. Also i have another issue. I want to create a widget within my Xcode project and then transfer it to my .net MAUI solution but it tells me i cannot. Do i need something specific to make it work or it is just my code? Thanks in advance!

1. What is the command that you have to use to enable it?
2. What type of widget are you trying to create?

**Comment by DgkanatsiosMasoutis** [comment]
1. This is the message i get every time i use open VS Code: 

```
Tools directory '/Users/MY-PATH/.dotnet/tools' is not currently on the PATH environment variable.

If you are using zsh, you can add it to your profile by running the following command:
 
cat << \EOF >> ~/.zprofile
Add .NET Core SDK tools
export PATH="$PATH:/Users/MY-PATH/.dotnet/tools"
EOF
 
And run `zsh -l` to make it available for current session.
 
You can only add it to the current session by running the following command:
 
export PATH="$PATH:/Users/MY-PATH/.dotnet/tools"
```


2. My widget is a view with a QR Code for the user to scan it using Timeline

**Comment by mcumming** [comment]
Ok, from the message you are receiving, it appears that you installed the .NET SDK using the install script and not the installer? If that is the case, the message is correct and you need to follow the instructions on adding the tools path to your profile so that you don't have to continually add it for every session.

As long as your widget is developed using C# it should work. `xcsync` cannot process a widget created in Xcode using ObjC, the only constructs we support syncing back from Xcode are Outlet and Actions.
